.travis.yml

```diff
@@ -1,39 +1,38 @@
-language: python
+language: 
+  - python
+
 python:
-  - 2.7
-  - 3.6
-  - 3.7
-  - 3.8
+  - "3.6"
+  - "3.7"
+  - "3.8"
```

Comment:
Yes, let's drop official Python 2.7 support in the next release. Could probably add Python 3.9 here too. I think it is live on conda forge.

Reply:
Agreed.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/629c6564272ba5f5988a145ecce3e617f27388e2.